go-controller/pkg/types/const.go

```diff
@@ -39,26 +39,33 @@ const (
 	// ACL Priorities
 
 	// Default routed multicast allow acl rule priority
-	DefaultRoutedMcastAllowPriority = "1013"
+	DefaultRoutedMcastAllowPriority = "10013"
 	// Default multicast allow acl rule priority
-	DefaultMcastAllowPriority = "1012"
+	DefaultMcastAllowPriority = "10012"
```

Comment:
upgrade implications here

Reply:
Good catch, do we have any plans to add an upgrade KIND job? From master HEAD -> branch HEAD for example?

Reply:
Actually come to think of it, it's even worse than only this commit. Whenever anybody (OCP) will upgrade from local gateway mode to shared gateway mode, the entire egress firewall setup will most likely break everything since in local gateway mode everything is implemented using routing rules, whereas in shared gateway mode it's ACL on the join switch...and egress firewall does not sync any of that.

Reply:
I am proposing (though it hurts given the effort its taken me to implement and constantly rebase) that we go for option 1.  OCP will move to shared gateway mode very soon, and as with any software version improvement, I think we can live with 4.7 having some rules which are sub-optimal, which when we move to shared gateway mode will be solved.  

Having this intermediary optimal local gateway mode will clutter the code with an over complicated sync function which will need to sync old local GW mode -> better local GW mode **and** better local GW mode -> shared GW mode. That's not worth the win we get from improving the rules.

I will write up a patch for syncing egress firewall setup from local to shared gateway mode.

Reply:
honestly I'm struggling a little to follow, but this seems reasonable to me. Just concerned about corner cases, leftover config, but if you are confident your newly added sync covers it then we should be good.

Reply:
Is "Remove inefficient inequality operator in firewall match" already done in shared gateway egress firewall?

Reply:
> Is "Remove inefficient inequality operator in firewall match" already done in shared gateway egress firewall?

It never had it. Only local gateway mode added match conditions where it filtered out traffic using the inequality operator. 

> honestly I'm struggling a little to follow

We can have a discussion concerning this on slack.